src/EnrollChoicesController.js

```diff
@@ -36,13 +40,11 @@ function (Okta, FormController, Enums, RouterUtil, FactorList,
     Form: {
       noCancelButton: true,
 
-      title: _.partial(Okta.loc, 'enroll.choices.title', 'login'),
```

Comment:
I don't see any special handling for the title, why do we need to change this?

Reply:
Okta.tpl takes allows us to pass an argument for which element we want to attach the template to. When i passed `$('.o-form-content')` for the subtitle element, it attached it as the first child.. So the order was subtitle -> title.. I couldn't find a way to fix this besides creating a template for title as well.. Is there a way to attach the template as a sibling instead?